omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1293,6 +1294,27 @@ def draw_scalebar(self, panel, region_width, page):
         scale_to_canvas = panel['width'] / float(region_width)
         canvas_length = pixels_length * scale_to_canvas
 
+        unit_symbols = {
+            "PICOMETER": {'symbol': "pm", 'microns': 0.000001},
```

Comment:
could those values not be retrieved from the model units?

Reply:
Done in f5db4b9